### ISSUE TITLE
Expo compatibility

### ISSUE THREAD

**Issue by fedxgibson** [FidelLimited/rn-sdk#82]
I am new to the Fidel ecosystem. I'm interested to know if this will work with expo. Thanks!

**Comment by blabz2007** [comment]
Yes it should work without any configuration as seen on [https://www.npmjs.com/package/fidel-react-native?activeTab=readme](https://www.npmjs.com/package/fidel-react-native?activeTab=readme)

Expo automatically sets a androidMinSdkVersion and iOS deployment target above what's required from the package.
And it's already using jitpack as well.

Fidel, you should update your documentation to explain this. This wasn't quick or easy to figure out and sent me down the rabbit hole of building custom plugins for Expo.
